spec/models/product_action_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe ProductAction, type: :model do
+  it 'is valid with valid attributes' do
+    expect(build(:product_action)).to be_valid
+  end
+  it 'is not valid without an action_type' do
+    expect(build(:product_action, action_type: nil)).not_to be_valid
```

Comment:
creo que no importa mucho, pero acá puedes usar los _matchers_ de rspec y hacer algo del tipo
`it { is_expected.to validate_presence_of(:action_type) }`

Reply:
buena, voy a leerme bien los matchers que tiene rspec, que no estoy muy familiarizado con la sintáxis 👌🏼